src/plugins/vis_type_timeseries/public/application/components/data_time_range_mode_label.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiFlexItem, EuiLink, EuiText, EuiToolTip } from '@elastic/eui';
+import { getUISettings } from '../../services';
+import { convertIntervalIntoUnit, isAutoInterval, isGteInterval } from './lib/get_interval';
+import { createIntervalBasedFormatter } from '../components/lib/create_interval_based_formatter';
+import { TIME_RANGE_DATA_MODES } from '../../../common/timerange_data_modes';
+import { PanelData } from '../../../common/types';
+
+const lastValueFormattedMessage = i18n.translate(
+  'visTypeTimeseries.dataTimeRangeModeLabel.lastValueMode',
+  {
+    defaultMessage: 'Last value',
+  }
+);
+
+interface DataTimeRangeModeLabelProps {
+  seriesData?: PanelData['data'];
+  panelInterval: number;
+  modelInterval: string;
+  modelTimeRangeMode: TIME_RANGE_DATA_MODES;
+}
+
+export const DataTimeRangeModeLabel = ({
+  seriesData,
+  panelInterval,
+  modelInterval,
+  modelTimeRangeMode,
+}: DataTimeRangeModeLabelProps) => {
+  const dateFormat = getUISettings().get('dateFormat');
+  const scaledDataFormat = getUISettings().get('dateFormat:scaled');
+
+  const getFormattedPanelInterval = () => {
+    const interval = convertIntervalIntoUnit(panelInterval, false);
+    return interval && `${interval.unitValue}${interval.unitString}`;
+  };
+
+  const getLastValueLabelWithTooltip = () => {
+    const formatter = createIntervalBasedFormatter(panelInterval, scaledDataFormat, dateFormat);
+    const lastBucketDate = formatter(seriesData![seriesData!.length - 1][0]);
+    const formattedPanelInterval =
+      (isAutoInterval(modelInterval) || isGteInterval(modelInterval)) &&
+      getFormattedPanelInterval();
+
+    const tooltipContent = (
+      <FormattedMessage
+        id="visTypeTimeseries.dataTimeRangeModeLabel.lastValueTooltip"
+        defaultMessage="Bucket: {bucket} {interval}"
+        values={{
+          bucket: lastBucketDate,
+          interval: formattedPanelInterval && (
+            <p>
+              <FormattedMessage
+                id="visTypeTimeseries.dataTimeRangeModeLabel.panelInterval"
+                defaultMessage="Interval: {formattedPanelInterval}"
+                values={{ formattedPanelInterval }}
+              />
+            </p>
+          ),
+        }}
+      />
+    );
+
+    return (
+      <EuiToolTip position="top" content={tooltipContent}>
+        <EuiLink href="#">{lastValueFormattedMessage}</EuiLink>
```

Comment:
This shouldn't be a link.

Reply:
Agree, let's replace to `EuiButtonEmpty`